src/Magnum/ImGuiIntegration/Widgets.h

```diff
@@ -29,7 +29,7 @@
 */
 
 /** @file
- * @brief Function @ref Magnum::ImGuiIntegration::image()
+ * @brief Functions @ref Magnum::ImGuiIntegration::image(), @ref Magnum::ImGuiIntegration::imageButton()
```

Comment:
Keep the `Function`, since I already have that everywhere else (consistency).

Reply:
Got it.